airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -50,6 +50,12 @@
   icon: alloydb.svg
   sourceType: database
   releaseStage: generally_available
+  allowedHosts:
+    hosts:
+      - "${host}"
+      - "*.cloud.google.com"
+      - "*.googleapis.com"
```

Comment:
Why would we need these in addition to the provided host or SSL tunnel?

<img width="675" alt="Screenshot 2023-02-28 at 2 19 17 PM" src="https://user-images.githubusercontent.com/303226/221994458-bc245802-9b82-4533-b91c-7b1fbe1cfcf0.png">

Reply:
The thinking is that hosts under `cloud.google.com` and `googleapis.com` are needed to access Google Services such as authentication tokens and storage. We do not explicitly create connections to these 2 domains from `source-alloydb` so I am not able to confirm if they are actually needed or not.

Reference: https://cloud.google.com/apis/docs/overview

Reply:
Granting access to all of google cloud seems a bit broad - a shady connector could be uploading things to GCS for example, which is the kind of thing we are trying to block with these firewalls.

AlloyDB seems to be using the postgres protocol, which I think means that it should only need access to the database host?

Reply:
Agree wildcards here are not ok, the bigger question is which domains do we need to whitelist access to when using cloud hosted databases? I will setup an AlloyDB instance to try this out.

By a shady connector, you mean in general, right? AlloyDB should go through a `db-sources` team review.

Reply:
ok, I tested with an AlloyDB database verified I was able to sync without needing wildcards on GCP domains. So I have removed them and the `allowedHost` property is the same postgres source. cc @evantahler @subodh1810 

`2023-03-03 06:02:48 [32mINFO[m i.a.w.p.DockerProcessFactory(create):128 - Creating docker container = source-alloydb-read-102-0-mfvec with resources io.airbyte.config.ResourceRequirements@5fd69a9e[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=] and _allowedHosts io.airbyte.config.AllowedHosts@14134dca[hosts=[10.41.128.7, 34.123.104.114, *.datadoghq.com, *.datadoghq.eu, *.sentry.io],additionalProperties={}]_`